app/models/air_pollution_prediction.rb

```diff
@@ -27,6 +27,23 @@ def initialize(
     @label = label
   end
 
+  TAG_COLOURS = {
+    1 => "bg-lime-400",
+    2 => "bg-green-400",
+    3 => "bg-lime-600",
+    4 => "bg-yellow-300",
+    5 => "bg-amber-200",
+    6 => "bg-yellow-500",
+    7 => "bg-orange-500",
+    8 => "bg-red-500",
+    9 => "bg-red-800",
+    10 => "bg-stone-700"
+  }
+
+  def daqi_indicator_colour_class
+    TAG_COLOURS.fetch(value)
+  end
+
```

Comment:
ah of course, i forgot about view components! I was wrestling with how awkward it would be to have that logic in a template but of course a view component is the obvious solution. thanks 💡